src/pages/Widget/Widget.tsx

```diff
@@ -40,8 +54,8 @@ function Content({
       hide_bg_tip: false,
     },
     isDescriptionTextShown: true,
-    liquidSplitPct: 50,
-    splitDisabled: false,
+    liquidSplitPct: endowment?.splitLiqPct ?? 50,
```

Comment:
on the widget configuration page, db values are only used as initial split values - but would be overriden by whatever user chooses in his widget configuration